### ISSUE TITLE
Using the Apple Pencil with scribble turned on results in characters being typed rather than handwritten. 

### ISSUE THREAD

**Issue by chrjohns** [daledesilva/obsidian_ink#35]
Using the Apple Pencil with scribble turned on results in characters being typed rather than handwritten.

**Comment by tripp396** [comment]
I’ve noticed this a couple times too and normally happens the first time I start a new handwriting section but I haven’t paid enough attention to narrow down the exact situation that causes it. It doesn’t happen like the other issue where tapping the icons does the same thing (I am on beta)

**Comment by daledesilva** [comment]
Oh, I just reread this and saw that it's "if the pen strays outside the handwriting section".

Is this only if you're starting a new stroke outside or is it also if you start inside but end up outside before raising the pencil?

@tripp396 is yours based on if the pen goes outside? Or anytime.

The writing section doesn't automatically get focus, so I imagine that if your first touch of the stylus is to do a fully, unbroken word, maybe it will use the scribble function because the focus is still in the markdown doc?

Whereas if the first interaction is a short stroke or a click on a button in the writing section, then the section would get focus quicker and you might not notice a scribble issue.

Mind testing that?

**Comment by tripp396** [comment]
@daledesilva You nailed it. Basically the handwriting section isn’t in focus and I did notice the cursor blinking out side of it. It kind of looks like its writing in the box but then it converts to a word via scribble.

Made it a habit to make sure I grab focus of the handwriting box before I start writing.

**Comment by daledesilva** [comment]
Sounds like one way to mitigate this will be to activate the handwriting
section based on hover.
That way, with a hover enabled stylus, the focus will always be on the ink
section before you touch pen to screen.

But it might create some confusing behaviour when you’re just using the pen
to do stuff outside.
Also would need to be an option that hover enables stylus owners could turn
on.

I don’t have a hover enabled pencil, but I can turn scribble on and see
what I can come up with. Won’t be high priority though as it sounds like
it’s still workable and there are a bunch of other parts I’m focussed on.

___________________________________
Dale de Silva
twitter: @daledesilva
projects: daledesilva.com

Does your business support a discussion about Australia Day?
Ask me about displaying your support on changeitourselves.com.au

**Comment by MikkelKappelPersson** [comment]
I'm am also an avid scribble user. I'm my case, writing turns into a scribble text 99 % of the time. This make the plugin unusable unless in turn off scribble. However, I use a 3rd party pen (adonit note +2). So I don't know if the functionality is slightly different.

**Comment by daledesilva** [comment]
I don't think it is different. I'm not sure there will be a way around this any time soon, unfortunately.

Perhaps there's a way to make an apple shortcut that turns off scribble when you open Obsidian?

**Comment by rbradleyc02** [comment]
avid scribble user as well and found another option for workaround...

if you switch into reading mode, you can still write in the handwriting section and scribble mode is no longer an issue

**Comment by codemaestro** [comment]
Great plugin!

> avid scribble user as well and found another option for workaround...
> if you switch into reading mode, you can still write in the handwriting section and scribble mode is no longer an issue

Something must have changed in the past year. In Reading mode, handwritten sections accept pen input, but the writing and drawing doesn't save. it would be much better for the plugin widget to disable Scribble automatically or provide a switch.

I like having access to Scribble if I need it and turn it off if I don't. It's inconvenient to switch to iPad Settings to disable Scribble when I want to add handwriting/drawings.

**Update**

I just watched your develop update videos and realized you switched the framework for v0.5.0. Looking forward to the updates!
